config-linux.md

```diff
@@ -196,6 +196,27 @@ For example, to run a new process in an existing container without updating limi
 
 Runtimes MAY attach the container process to additional cgroup controllers beyond those necessary to fulfill the `resources` settings.
 
+### Cgroup ownership
+
+Runtimes MAY change (or cause to be changed) the owner of the
+container's cgroup to the host uid that maps to uid 0 in the
+container's user namespace, if cgroups v2 is in use.
+
+Runtimes MUST NOT change the ownership of container cgroups when
+cgroups v1 is in use.  Cgroup delegation is not secure in cgroups
+v1.
+
+Runtimes MUST only change the ownership of the container's cgroup
```

Comment:
By default runtime MUST NOT change anything.
We need a new config field to opt-in to this.

Reply:
That would put some existing implementations in immediately violation of the spec - `crun`, at least, already chowns the cgroup.  I'd appreciate more perspectives on this proposal.

Reply:
But if we adopt the crun behavior, that would put runc in violation of the spec.

As runc is the reference implementation of the spec,  I suggest modifying this PR and crun implementation.

Reply:
The proposal does not say runtimes MUST chown the cgroup - only that if they do, they MUST chown only the particular files mentioned.  I will update the language to clarify.

Reply:
by default `/sys/fs/cgroup` is mounted as read-only so in practice the cgroup ownership doesn't change much in what the container is allowed to do.  In order to use the `/sys/fs/cgroup` the mount must be writeable.

Maybe we can do the chowning when `/sys/fs/cgroup` is mounted as `rw`?  Otherwise it is unhelpful to have `/sys/fs/cgroup` writeable from an user not present in the current user namespace.

Reply:
> Maybe we can do the chowning when /sys/fs/cgroup is mounted as rw?

SGTM :+1:
I think when the mount is RW && cgroup is v2, we can safely say that the runtime `SHOULD` chown the cgroup.

Reply:
@kolyshkin @cyphar PTAL

Reply:
@giuseppe @AkihiroSuda thanks for your suggestion.  I updated the text to specify that the runtime SHOULD chown the cgroup when the cgroupfs is mounted RW (and is cgroups v2), otherwise MUST NOT.